source/Ship.cpp

```diff
@@ -663,10 +663,10 @@ void Ship::FinishLoading(bool isNewInstance)
 			string warning = (!isYours && !variantName.empty()) ? "variant \"" + variantName + "\"" : modelName;
 			if(!name.empty())
 				warning += " \"" + name + "\"";
-			warning += ": outfit \"" + outfit->Name() + "\" installed as a ";
```

Comment:
Since this is for the error log, use the actual name instead of the display name.

Reply:
done